src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+* {
+  padding: 0;
   margin: 0;
```

Comment:
Avoid using the universal selector '*' for styling as it impacts performance. Set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for browser. Refer to the checklist item: 'Don't use * selector (it impacts performance)'.